osp/serializers/form.py

```diff
@@ -2,12 +2,18 @@
 
 from osp.models import Form
 
-class FormSerializer(serializers.ModelSerializer):
 
+class FormSerializer(serializers.ModelSerializer):
     class Meta:
-        
+
         model = Form
         fields = [
-            'id', 'name', 'description', 'published_status',
-            'questions', 'target_user', 'created_on', 'updated_on'
+            "id",
+            "name",
+            "description",
+            "published_status",
+            "questions",
+            "target_user",
+            "created_on",
+            "updated_on",
```

Comment:
can we can use `__all__` here?

Reply:
will created_on and updated_on will also be imported?